src/storage/test/DeleteVertexTest.cpp

```diff
@@ -22,7 +22,7 @@ TEST(DeleteVertexTest, SimpleTest) {
     std::unique_ptr<kvstore::KVStore> kv(TestUtils::initKV(rootPath.path()));
     // Add vertices
     {
-        auto* processor = AddVerticesProcessor::instance(kv.get(), nullptr, nullptr);
+        auto* processor = AddVerticesProcessor::instance(kv.get(), nullptr, nullptr, nullptr);
```

Comment:
I have a question? How do I construct a indexManager for testcase?

Reply:
doing: Adhoc Index Manager. Be similar to Adhoc Schema Manager.